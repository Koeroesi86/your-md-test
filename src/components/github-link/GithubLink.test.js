import React from 'react'
import { shallow } from 'enzyme'
import GithubLink from './GithubLink'

describe('The GithubLink component', () => {
  it('renders', () => {
    const result = shallow(<GithubLink/>);

    expect(result).toMatchSnapshot();
  });
});
