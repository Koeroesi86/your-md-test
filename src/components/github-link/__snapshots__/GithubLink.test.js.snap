// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The GithubLink component renders 1`] = `
<a
  className="App-github-link"
  href="https://github.com/Koeroesi86"
  rel="noopener noreferrer"
  target="_blank"
  title="Visit my Github"
>
  <img
    alt="Github"
    className="App-github-link-image"
    src="github.png"
  />
</a>
`;
